addons/getLoadoutsFromUnits/functions/fnc_addMedicItems.sqf

```diff
@@ -0,0 +1,106 @@
+params ["_unit", "_type"];
+
+private _log = [];
+private _items = [];
+private _loadout = getUnitLoadout _unit;
+
+{
+   if (!((_loadout select _x) isEqualTo []) && !((_loadout select _x select 1) isEqualTo [])) then {
+      _items append (_loadout select _x select 1);
+   };
+}forEach [3,4,5];
+
+{
+   private _countOfSameItem = 0;
+   private _index = _forEachIndex;
+   {
+      _x params ["_newItem", "_amount"];
+      {
+         if ((_x select 0) == _newItem) then {
+            _countOfSameItem = _countOfSameItem +1;
+         };
+      }forEach _items;
+      
+      private _rest = _amount - _countOfSameItem;
+      if (_rest > 0) then {
+         for "_i" from 1 to _rest do {
+            _fits = _unit canAddItemToBackpack _newItem;
+            if (_fits) then {
+               _unit addItemToBackpack _newItem;
+            }else{
+               _log pushBack _newItem;
+            };
+         };
+      };
+   }forEach _x;
+}forEach ([
```

Comment:
missing spaces

Reply:
Thats an ACE thing, here I can do it as I see fit :P